packages/mrm/src/__tests__/index.spec.js

```diff
@@ -106,6 +123,61 @@ describe('tryResolve', () => {
 	});
 });
 
+describe('installGlobalPackage', () => {
+	it('should resolve to true if able to install the package', async () => {
+		spawn.mockReturnValueOnce({ on: spawnOnErrorMock });
+		const pkgName = String(Date.now());
+		const result = await installGlobalPackage(pkgName);
+		expect(spawn).toBeCalledWith('npm', ['install', '--global', pkgName], {
+			stdio: 'inherit',
+		});
+		expect(spawnOnErrorMock.mock.calls[0][0]).toBe('error');
+		expect(spawnOnErrorMock.mock.calls[0][1]).toBeInstanceOf(Function);
+		expect(spawnOnCloseMock.mock.calls[0][0]).toBe('close');
+		expect(spawnOnCloseMock.mock.calls[0][1]).toBeInstanceOf(Function);
+		expect(result).toBeTruthy();
+	});
+
+	it('should reject if unable able to install the package', async () => {
+		spawn.mockReturnValueOnce({ on: spawnOnErrorMock });
+		const error = Date.now();
+		spawnOnErrorMock.mockImplementation((_, cb) => {
+			cb(error);
+		});
+		spawnOnCloseMock.mockImplementation(() => {});
+		const pkgName = String(Date.now());
+
+		try {
+			await installGlobalPackage(pkgName);
+			throw new Error('Should have thrown');
```

Comment:
I'm not sure how this is supposed to work.  I think promise matchers is what you need here:

https://github.com/sapegin/jest-cheat-sheet/blob/master/Readme.md#promise-matchers-jest-20